whats_fresh/whats_fresh_api/views/entry/vendors.py

```diff
@@ -219,7 +220,7 @@ def vendor_list(request):
         'message': message,
         'new_url': reverse('new-vendor'),
         'new_text': "New Vendor",
-        'title': "All Vendors",
+        'title': get_fieldname.get_fieldname('vendors', 'vendors'),
```

Comment:
@tschuy I have 2 points to make regarding this - 

Firstly. it works, however I am not completely sure if this is the correct way to do this.

Secondly, earlier the title was **All Vendors** and now it will be just the alternate form name as chosen from the admin by the user or **Vendors** if not chosen by the user-- is that fine?